articles/azure/azs-how-create-load-balancer.md

```diff
@@ -92,23 +90,29 @@ In this section we will create a virtual network and two virtual machines which
 
 4. In the **Create virtual network** blade, enter the following information:
 
-   - **Name** - The name of the virtual network.
+    - *Basics*
+
+        - **Subscription** - This is your UKCloud for Microsoft Azure subscription and will be pre-populated by default.
+
+        - **Resource Group** - Select an existing resource group, or create a new one by clicking the **Create new** link and then typing a name for your new resource group in the pop out window.
 
-   - **Address Space** - The virtual network's address range in CIDR notation (for example, 192.168.1.0/16).
+        - **Name** - The name of the virtual network.
 
-   - **Subscription** - This is your UKCloud for Microsoft Azure subscription and will be pre-populated by default.
+        - **Region** - This will be `frn00006`, which is the location of the Azure Stack Hub.
 
-   - **Resource Group** - Select an existing resource group, or create a new one by clicking the **Create new** link and then typing a name for your new resource group in the pop out window.
+      ![Create new virtual network - Basics](images/azs-browser-create-virtual-network.png)
 
-   - **Location** - This will be `frn00006`, which is the location of the Azure Stack Hub.
+    - *IP Addresses*
 
-   - **Subnet Name** - The name of the first subnet within the virtual network.
+        - **IPv4 address space** - The virtual network's address range in CIDR notation (for example, 192.168.1.0/16).
 
-   - **Address Range** - The subnet's address range in CIDR notation (for example, 192.168.1.0/24). It must be contained by the address space of the virtual network. The address range of a subnet which is in use can't be edited.
+        - **Subnet name** - The name of the first subnet within the virtual network.
 
-     ![Create new virtual network](images/azs-browser-create-virtual-network.png)
+        - **Subnet address range** - The subnet's address range in CIDR notation (for example, 192.168.1.0/24). It must be contained by the address space of the virtual network. The address range of a subnet which is in use can't be edited.
 
-5. Click **Create**.
+      ![Create new virtual network - IP Addresses](images/azs-browser-create-virtual-network2.png)

```

Comment:
should the subscription be greyed out like in other screenshots?

Reply:
Yes it should, good spot. I've updated the image.